crates/ruma-common/src/thirdparty.rs

```diff
@@ -255,6 +259,27 @@ pub struct ThirdPartyIdentifier {
     pub added_at: MilliSecondsSinceUnixEpoch,
 }
 
+impl Borrow<str> for ThirdPartyIdentifier {
+    fn borrow(&self) -> &str {
+        &self.address
+    }
+}
+
+#[cfg(not(test))]
```

Comment:
Why is it not implemented for tests?

Reply:
> Why is it not implemented for tests?

Tests use PartialEq to assert correct (de)serialization. In practice however the timestamp fields are never gonna be equal nor do they serve as identifiers.

Reply:
Oh I didn't see the derive macro above. In that case just change the tests and compare field by field? It seems confusing to me to have different implementations here.

Reply:
The coffee machine broke it seems.